src/test/frontend/handler/editor_test.cljs

```diff
@@ -238,4 +245,20 @@
      (is (= "b12 #foo" (:block/content updated-block)) "Content updated correctly")
      (is (= prev-path-refs
             (set (map :block/name (:block/path-refs updated-block))))
-         "Path-refs remain the same"))))
\ No newline at end of file
+         "Path-refs remain the same"))))
+
+(deftest save-block
+  (testing "Saving blocks with and without properties"
+    (test-helper/load-test-files [{:file/path "foo.md"
+                                   :file/content "# foo"}])
+    (let [repo test-helper/test-db
+          block-uuid (:block/uuid (model/get-block-by-page-name-and-block-route-name repo "foo" "foo"))
+          _ (let [_ (editor/save-block! repo block-uuid "# bar")
```

Comment:
It's unusual for lets to be nested in each other and to not have bodies. Gonna clean this up